tests/system_tests/test_launch/test_launch_run.py

```diff
@@ -36,45 +36,41 @@ def test_run_use_job_env_var(runner, relay_server, test_settings, user, wandb_in
         assert use_count == 1
 
 
-def test_run_in_launch_context_with_multi_config_env_var(
-    runner, test_settings, monkeypatch, wandb_init
-):
+def test_run_in_launch_context_with_multi_config_env_var(runner, monkeypatch, user):
```

Comment:
and here

Reply:
i don't think i can here since we do wandb.init